modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -308,49 +334,57 @@ public OAuthConsumerAppDTO createApplication(OAuthConsumerAppDTO appDTO, String
-		ServiceProvider serviceProvider = appMgtclient.getApplication(SERVICE_PROVIDER_NAME);
-		ClaimConfig claimConfig = getClaimConfig();
-		serviceProvider.setClaimConfig(claimConfig);
-		appMgtclient.updateApplicationData(serviceProvider);
+		claimConfiguration.addClaimMappingsItem(getClaimMapping(GIVEN_NAME_CLAIM_URI));
+		claimConfiguration.addRequestedClaimsItem(getRequestedClaim(GIVEN_NAME_CLAIM_URI));
+
+		claimConfiguration.addClaimMappingsItem(getClaimMapping(COUNTRY_CLAIM_URI));
+		claimConfiguration.addRequestedClaimsItem(getRequestedClaim(COUNTRY_CLAIM_URI));
+
+		claimConfiguration.addClaimMappingsItem(getClaimMapping(customClaimURI1));
+		claimConfiguration.addRequestedClaimsItem(getRequestedClaim(customClaimURI1));
+
+		claimConfiguration.addClaimMappingsItem(getClaimMapping(customClaimURI2));
+		claimConfiguration.addRequestedClaimsItem(getRequestedClaim(customClaimURI2));
+
+		return claimConfiguration;
 	}
 
-	private ClaimConfig getClaimConfig() {
-		ClaimConfig claimConfig = new ClaimConfig();
-		ClaimMapping emailClaimMapping = getClaimMapping(EMAIL_CLAIM_URI);
-		ClaimMapping givenNameClaimMapping = getClaimMapping(GIVEN_NAME_CLAIM_URI);
-		ClaimMapping countryClaimMapping = getClaimMapping(COUNTRY_CLAIM_URI);
-		ClaimMapping customClaimMapping1 = getClaimMapping(customClaimURI1);
-		ClaimMapping customClaimMapping2 = getClaimMapping(customClaimURI2);
-		claimConfig.setClaimMappings(new org.wso2.carbon.identity.application.common.model.xsd
-				.ClaimMapping[]{emailClaimMapping, givenNameClaimMapping, countryClaimMapping, customClaimMapping1,
-				customClaimMapping2});
-		return claimConfig;
+	private ClaimMappings getClaimMapping(String claimUri) {
+		ClaimMappings claim = new ClaimMappings().applicationClaim(claimUri);
+		claim.setLocalClaim(new org.wso2.identity.integration.test.rest.api.server.application.management.v1.model.Claim().uri(claimUri));
+		return claim;
 	}
 
-	private ClaimMapping getClaimMapping(String claimUri) {
-		Claim claim = new Claim();
-		claim.setClaimUri(claimUri);
-		ClaimMapping claimMapping = new ClaimMapping();
-		claimMapping.setRequested(true);
-		claimMapping.setLocalClaim(claim);
-		claimMapping.setRemoteClaim(claim);
-		return claimMapping;
+	private RequestedClaimConfiguration getRequestedClaim(String claimUri) {
+		RequestedClaimConfiguration requestedClaim = new RequestedClaimConfiguration();
+		requestedClaim.setClaim(new org.wso2.identity.integration.test.rest.api.server.application.management.v1.model.Claim().uri(claimUri));
+		return requestedClaim;
 	}
 
 	/**
 	 * Send post request with parameters
-	 * @param client
-	 * @param urlParameters
-	 * @param url
-	 * @return
-	 * @throws ClientProtocolException
-	 * @throws java.io.IOException
+	 * @param client HttpClient
+	 * @param urlParameters url parameters
+	 * @param url endpoint
+	 * @return HttpResponse
+	 * @throws ClientProtocolException ClientProtocolException
+	 * @throws java.io.IOException java.io.IOException
```

Comment:
Shall we fix the formatting by adding missing fullstops?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096